pyproject.toml

```diff
@@ -216,6 +217,7 @@ files = [
     "frappe/types/DF.py",
     "frappe/types/docref.py",
     "frappe/types/frappedict.py",
+    "frappe/types/filter.py",
```

Comment:
This also ensures it will be automatically picked up by the Typing Check in CI, if modified